features/userprofile/shared/src/main/kotlin/io/element/android/features/userprofile/shared/UserProfileState.kt

```diff
@@ -14,21 +14,22 @@
  * limitations under the License.
  */
 
-package io.element.android.features.roomdetails.impl.members.details
+package io.element.android.features.userprofile.shared
 
 import io.element.android.libraries.architecture.AsyncAction
 import io.element.android.libraries.architecture.AsyncData
 import io.element.android.libraries.matrix.api.core.RoomId
+import io.element.android.libraries.matrix.api.core.UserId
 
-data class RoomMemberDetailsState(
-    val userId: String,
+data class UserProfileState(
+    val userId: UserId,
     val userName: String?,
     val avatarUrl: String?,
     val isBlocked: AsyncData<Boolean>,
     val startDmActionState: AsyncAction<RoomId>,
     val displayConfirmationDialog: ConfirmationDialog?,
     val isCurrentUser: Boolean,
-    val eventSink: (RoomMemberDetailsEvents) -> Unit
+    val eventSink: (UserProfileEvents) -> Unit
```

Comment:
Room member detail and User profile now have the same actions and rendering, so the state is common, but at some point we may want to add extra data here, like a `RoomId?`, and extra action like `Jump to Read Receipt`, only if a room is there.